.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+default_language_version:
+    python: python3.8
```

Comment:
Please undo these changes. Py3 should be the default for everyone anyway, now that Py2 is deprecated. Also, it fails the tests

Reply:
done!